migrations/versions/856b03419a0b_created_user_model.py

```diff
@@ -0,0 +1,36 @@
+"""created User model
```

Comment:
as a rule of thumb, use one migration per PR. this way it will be easier to rollback your change if needed